apps/nameguard.io/src/components/molecules/HeroStartCommand.tsx

```diff
@@ -1,7 +1,7 @@
 "use client";
 
 import { useEffect, useState } from "react";
-import { Tooltip } from "@namehash/nameguard-react";
+import { Tooltip } from "@namehash/namekit-react";
```

Comment:
This is broken. Please ensure you test all changes yourself before asking for a PR review.

![CleanShot 2024-08-25 at 17 35 42](https://github.com/user-attachments/assets/80019dd5-470b-442f-88e7-302819914917)

Reply:
@lightwalker-eth fixed